client/src/Alerts.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { Alert } from 'react-bootstrap'
+
+class Alerts extends React.Component {
+  render () {
+    return (
+      <div>
+        {displayAlert(this.props.state)}
+      </div>
+    )
+  }
+}
+
+// TODO: Set a 2 second timer to display alert before it disapears
+function displayAlert (state) {
+  if (state.status === 1) {
+    return (
+      <Alert variant='danger'>
+      You have already created an account with this email! Please check your email to login.
+      </Alert>
+    )
+  } else if (state.status === 409) {
```

Comment:
I feel like SignupAlerts should be a separate component from FormAlerts. Otherwise, it's possible to accidentally have a SignupAlert in the AddFriend page and vice versa. By separating them, it's easier to edit/reason about. In addition, it makes it possible to have two different messages for the same status code.

If you're worried about duplicating logic, then that's a good sign that you could benefit from making a general purpose BaseAlert component and then making `class SignupAlert extends BaseAlert` and `class FormAlert extends BaseAlert`. If there's minimal duplication of code, then it's probably fine to just make two separate components.